src/atomate2/common/flows/pheasy.py

```diff
@@ -0,0 +1,374 @@
+"""Flows for calculating phonons."""
```

Comment:
```suggestion
"""Flows for calculating phonons with the pheasy code."""
```

Reply:
Maybe we could make a module called "common/flows/phonons/pheasy.py" and also move the phonons.py part there and rename it to phonopy.py. We then need to ensure backward compatibility via the __init__.py in the phonons module. Same for jobs/schemas etc.

Reply:
We could also allow for an easy switch between phonopy and pheasy in the current workflow as many parts of the workflow are quite similar.

Reply:
I agree, creating a common/flows/phonons/pheasy.py and phonopy.py